File: code/MATLAB/Files/Diffusion/initBar.m
function bar = initBar(m, n,hotSites, coldSites)
% INITBAR return m-by-n bar initialized with all ambient temperatures
% except for hot and cold sites
global AMBIENT

bar = AMBIENT * ones(m, n);
bar = applyHotCold(bar, hotSites, coldSites);

File: code/R/R_Computational_Toolbox_Files/Diffusion_R/initBar.R
#Initialize the bar
initBar <-function(m,n,hotSites,coldSites){
utils::globalVariables(c("AMBIENT"))
bar <-AMBIENT*(matrix(c(rep(1,m*n)), nrow = m))
bar = applyHotCold(bar, hotSites,coldSites)
return(bar)}

